linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithDnf.java

```diff
@@ -43,25 +43,28 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
 		 * ubi8 has DNF installed
 		 */
 
-		String containerRegistry = "";
+		String quayIoDefaultRegistry = "quay.io/";
+		String dockerDefaultRegistry = "";
 
 		if (System.getenv("containerRegistry") == null) {
-			System.out.println("Using docker.io as the default container registry");
+			System.out.println("Using docker.io and quay.io as the default container registry");
```

Comment:
I thought docker.io is already gone completely?

Reply:
I'm not sure what you mean, Docker still maintains the DockerHub and is the default for most container runtimes. CentOS does not currently publish to dockerhub